README.md

```diff
@@ -61,7 +61,7 @@ Full web-based HTML documentation for this app can be found over on the [Nautobo
 - [Administrator Guide](./docs/admin/) - How to Install, Configure, Upgrade, or Uninstall the App.
 - [Developer Guide](./docs/dev/) - Extending the App, API Reference, Contribution Guide.
 - [Release Notes / Changelog](./docs/admin/release_notes)
-- [Frequently Asked Questions](./docs/user/app_faq)
+- [Frequently Asked Questions](./docs/user/app_faq.md)
```

Comment:
@cmsirbu what is the way to address this?

Reply:
The best way is going to be to use full links here to the docs once they're hosted in the final location. It's part of the plan to reduce duplication (README.md and docs/index.md are the same content right now, but with different links). I'll fix this in #300 today.

Reply:
@cmsirbu I can just close this PR then correct?

Reply:
Sounds good, I've pushed the changes to #300.